cou/apps/app.py

```diff
@@ -118,6 +117,16 @@ def decorator(application: type[OpenStackApplication]) -> type[OpenStackApplicat
         return decorator
 
 
+@dataclass
+class ApplicationUnit:
+    """Representation of singe JujuUnit of application."""
```

Comment:
```suggestion
    """Representation of a singe unit of an application."""
```
I don't think `JujuUnit` is approprite here because it doesn't contain the object; instead, it only provides a handful attributes of a unit

Reply:
ok